shell/components/GrowlManager.vue

```diff
@@ -114,6 +114,11 @@ export default {
 
     @media only screen and (min-width: map-get($breakpoints, '--viewport-7')) {
       width: 350px;
+
+      // extra padding on the right so that the close X doesn't overlap the text
+      .growl-text {
+        padding-right: 30px !important;
```

Comment:
`growl-text` does already contain padding, so that should be updated instead of of an `!important` here. however it looks like the existing padding works fine with long text. was there a specific scenario you hit that it didn't?

![image](https://user-images.githubusercontent.com/18697775/195614453-8119aeef-75ab-418e-a90b-59a93386624f.png)

Reply:
maybe it was something that was fixed in these months already... going to remove the padding change